src/data/security-quiz.ts

```diff
@@ -8,6 +8,26 @@ const securityQuiz = [
     Explanation: "There is no attack model named CVA (Chosen vector attack).",
     Link: "https://en.wikipedia.org/wiki/Attack_model"
   },
+  {
+      Question:
+    "In the context of cybersecurity, what is the 'zero-day' vulnerability, and why is it particularly dangerous?",
+    Answer: "A vulnerability that is exploited before it is publicly known, making it unpatched and dangerous.",
+    Distractor1: "A vulnerability that exists for zero days, posing no threat.",
+    Distractor2: "A vulnerability with a zero percent likelihood of exploitation.",
+    Distractor3: "A vulnerability that can only be patched on the first day of discovery.",
+    Explanation: "Zero-day vulnerabilities are dangerous because they are exploited by cybercriminals before security patches are available, leaving systems exposed.",
+    Link: "https://en.wikipedia.org/wiki/Zero-day_(computing)"
+  },
+  {
+    Question:
+    "What is the purpose of the SQL HAVING clause in a query?",
+    Answer: "To filter the results of an aggregate function",
+    Distractor1: "To sort the results in descending order",
+    Distractor2: "To join multiple tables",
+    Distractor3: "To create a new table",
+    Explanation: "HAVING is used with GROUP BY to filter aggregated data based on a condition.",
+    Link: "https://en.wikipedia.org/wiki/Having_(SQL)"
+  },
```

Comment:
this is the security quiz and SQL questions should not be here
please remove this from this file